Sass/style.css

```diff
@@ -0,0 +1,555 @@
+}
+
+.home-page .home-buttons-div button {
+  margin-top: 15px;
+  width: 90%;
+  min-height: 60px;
+  text-align: center;
+  font-size: 20px;
+  color: rgb(255, 255, 255);
+  background-color: rgb(255, 0, 0);
+  border-radius: 25px;
+  margin-bottom: 20px;
+}
+
+.copy {
+  margin-top: 140px;
+  float: left;
+  font-weight: 700;
+}
+
+.copy b {
+  font-size: 30px;
+}
+
+.about-us {
+  display: flex;
+  flex-direction: column;
+  padding-top: 30px;
+  align-items: center;
+  background-image: url(../images/abt-bg.jpg);
+  width: 100%;
+}
+
+.about-us p {
+  display: flex;
+  width: 100%;
+  padding: 10px;
+  margin-top: 30px;
+  margin-bottom: 80px;
+  background-color: #fff;
+  box-shadow: 0 2px 8px rgba(104, 102, 102, 0.895);
+  min-height: 100px;
+}
+
+.about-us h1 {
+  display: flex;
+  float: left;
+  font-size: 50px;
+  color: #ec5242;
+}
+
+.about-us-descrptn {
+  display: flex;
+  justify-content: center;
+  background-color: white;
+  padding-left: 10px;
+  padding-bottom: 10px;
+  height: auto;
+}
+
+.courses p {
+  display: flex;
+  width: 85%;
+  color: white;
+}
+
+.about-us-descrptn .inner p {
+  display: flex;
+  width: 90%;
+}
+
+.joinUs-section h2 {
+  display: flex;
+  color: white;
+}
+
+.about-us-descrptn .inner h2 {
+  display: flex;
+
+  /* font-size: 20px; */
+}
+
+.courses-section {
+  display: flex;
+  padding-top: 20px;
+  flex-direction: column;
+  color: #fff;
+  justify-content: center;
+  align-items: center;
+  background-color: #212529;
+  width: 100%;
+}
+
+.courses-section h1 {
+  text-decoration-line: underline;
+}
+
+.course-links {
+  display: flex;
+  flex-wrap: wrap;
+  row-gap: 50px;
+  column-gap: 15px;
+  background-image: url(../images/program-back.png);
+  background-color: #212529;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+  padding-bottom: 30px;
+  padding-top: 40px;
+  max-width: 100%;
+  min-height: 300px;
+}
+
+.nav-menu.visible li :hover {
+  text-decoration-line: underline;
+}
+
+#feedback :nth-child(n+9) {
+  display: none;
+}
+
+.courses a {
+  display: block;
+  margin: 10px;
+  color: rgb(244, 14, 14);
+}
+
+footer .lets-connect a {
+  text-decoration-line: none;
+  display: inline-block;
+  padding-left: 10px;
+  margin-bottom: 20px;
+}
+
+.top-header li a {
+  color: white;
+}
+
+.courses a:hover {
+  transform: scale(1.15);
+  text-decoration-line: underline;
+}
+
+.courses {
+  color: white;
+  display: flex;
+  row-gap: 5px;
+  padding: 10px;
+  flex-direction: column;
+  justify-content: center;
+  background-color: rgba(255, 255, 255, 0.1);
+  box-sizing: border-box;
+  margin-right: 15px;
+  transform: scale(1.15);
+  align-items: center;
+  height: 240px;
+  max-width: 150px;
+}
+
+.courses:active {
+  border: #fff solid 1px;
+}
+
+.feedback-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  min-height: 700px;
+  width: 100%;
+  padding: 10px;
+}
```

Comment:
Great job @degisew 🤝

- [ ] As the previous reviewers pointed out, kindly make your design a replica of the design provided.
- Your design
![speakers-yours](https://user-images.githubusercontent.com/63917725/183267181-44e7484a-95ec-4bb8-ba7e-a357faf63f51.PNG)

- Given design
![given speakers](https://user-images.githubusercontent.com/63917725/183267186-ef5ab829-d5c9-40db-a77c-56770aeab48a.PNG)